### PR TITLE
PR: Expand Tutorial Instructions #42

### PR DESCRIPTION
**Pull request by nelsonic**
This PR builds upon @SimonLab's superb work expanding and some cases trimming the instructions. #42 
Also adds some UI Stylin' using **`Tailwind`** CSS for practice #4 

Before:
image

After:
liveview-chat-with-tailwind-css

Thanks! 🌻

### CODE REVIEW

**Comment by nelsonic** [comment]
This is taking longer than I had hoped ... ⏳ 
Each time I sit down to try and work on this I am distracted by _someone/something_ ... 🙃 
This morning it was: https://github.com/iteles/blw-baby/pull/32 which I felt deserved my attention. 💭 

Taking my laptop with me to the wedding this weekend.
if there is a quiet period in the morning when everyone else is "recovering" and **`A+++`** is still sleeping ...
I will try and continue making progress on it. 🤞

**Comment by nelsonic** [comment]
Re-added the Presence, but in the Nav:
image

Not going to spend any more time on this. I think the tutorial is where it needs to be. ✅ 
We can move on to building our App!! 💭

**Comment by nelsonic** [comment]
@SimonLab Apologies for this being a bit longer than I would have hoped. 🙃 
Not urgent to review. But when you can. Thanks. 🙏

**Comment by nelsonic** [comment]
Link to read the `README.md`: https://github.com/dwyl/phoenix-liveview-chat-example/tree/expand-instructions-issue-%2342

File: README.md [hunk 1]
```diff
@@ -1,59 +1,163 @@
...
+so we're excluding them from our app.
+You can learn more about creating
+new Phoenix apps by running:
+`mix help phx.new`
 
-Run `mix deps.get` to retrieve the dependencies, then make sure you have
-the `liveview_chat_dev` Postgres database available.
-You should now be able to start the application with `mix phx.server`:
+Run `mix deps.get` to retrieve the dependencies.
+then make create the
```
Comment (SimonLab) [inline README.md:115]:
```suggestion
then create the
```

File: README.md [hunk 2]
```diff
@@ -1,59 +1,163 @@
...
+The database for LiveviewChat.Repo has been created
 
+14:20:19.71 [info]  Migrations already up
+```
 
-```elixir
-scope "/", LiveviewChatWeb do
-  pipe_through :browser
+Once that command succeeds 
+You should now be able to start the application with
```
Comment (SimonLab) [inline README.md:132]:
```suggestion
You should now be able to start the application
```

File: README.md [hunk 3]
```diff
@@ -68,13 +172,25 @@ defmodule LiveviewChatWeb.MessageLive do
...
+
+A **`LiveView` controller** requires 
+the functions **`mount/3`** and **`render/1`** to be defined. <br />
+To keep the controller simple the **`mount/3`**
+is just returning the `{:ok, socket}` tuple
+without any changes.
+The **`render/1`** 
+invokes 
+`LiveviewChatWeb.MessageView.render/2` (included with `Phoenix`)
+which renders the **`message.html.heex`** `template`
```
Comment (SimonLab) [inline README.md:189]:
```suggestion
which renders the **`messages.html.heex`** `template`
```

File: README.md [hunk 4]
```diff
@@ -1015,7 +1336,8 @@ with the "presence_diff" event.
 The `handle_info` function catches the `presence_diff` event and reassigns to the socket
 the `presence` value with the result of the `get_presence_names` function call.
 
-To display the names we add the following in `lib/liveview_chat_web/templates/message/message.html.heex`
+To display the names we add the following in the
+`lib/liveview_chat_web/templates/message/message.html.heex`
```
Comment (SimonLab) [inline README.md:1340]:
```suggestion
`lib/liveview_chat_web/templates/message/messages.html.heex`
```

**Review by SimonLab — COMMENTED** [review]
I've spotted a few typos to fix.
I've also noticed the login button is still displayed when the user is authenticated (on the name, message section)

**Comment by nelsonic** [comment]
@SimonLab please take another look when you're back at your desk on Monday. Thanks! ☀️

**Review by SimonLab — APPROVED** [review]
Looks good, thanks for the updates :+1:

**Pull request merged**